quiz-app/src/assets/translations/hin.json

```diff
@@ -694,7 +694,7 @@
 								"isCorrect": "false"
 							},
 							{
-								"answerText": "उनके बीच एक बैटन गुजरते हुए एक दौड़ चलाने के लिए धावक",
+								"answerText": "धावकों को दौड़ मे उनके बीच  एक बैटन को पारित करते हुये दौड़ने को",
```

Comment:
I am sorry but I am not getting the meaning of this option.

"धावकों को दौड़ मे उनके बीच एक बैटन को पारित करते हुये दौड़ने को"

Kindly share the English option in reply to this comment for understanding it better.

Reply:
Runners to run a race passing a baton between them

Reply:
I am not really sure whether this option is anywhere related to the IoT.

I would like to see the question in the English JSON file for my better understanding.

Reply:
This option is not related to IoT. I t gives another meaning of relay.

Reply:
Okay, Then it's fine.

Sorry for my misunderstanding.

Reply:
@Anurag-0-1-A - is this now ready to merge?

Reply:
Yes